### ISSUE TITLE
Author profiles aren't accessible in multlingual setup

### ISSUE THREAD

**Issue by digitalcraftsman** [spf13/hugo#2469]
If I add the configuration for another language then the author profiles can't be accessed inside the templates. `.Site.Authors` for example remains an empty list. Have a look on the "Authors" header on the homepage.

The author profiles return if I [comment out](https://github.com/digitalcraftsman/hugo-multilingual/blob/master/config.toml#L45-L57) the config for the German translation.

**Comment by bep** [comment]
@derekperkins

**Comment by derekperkins** [comment]
I started my PR long before languages became a thing, and to be honest, I don't know anything about the implementation. It's just loading from data files inside `data/_authors`. Is that path being redirected somehow?

On that note, right now there isn't any support inside of author profiles for anything multi-lingual. How would you anticipate wanting to interact with it? Looking at the setup, it seems like most fields would be redundant if you nested them inside languages, but that might be the easiest way to go about it. Obviously I'd prefer to mimic however multi-lingual is handled elsewhere in Hugo.

```
[languages.en]

given_name      = "Derek"
family_name     = "Perkins"
display_name    = "Derek Perkins"
thumbnail       = "img link"
image           = "img link"
short_bio       = "super short bio"
bio             = "Here's Derek's awesome bio"
email           = "derek@email.com"

[social]
    facebook    = "derekperkins"
    github      = "derekperkins"
    twitter     = "derek_perkins"
    googleplus  = "DerekPerkins1"

[params]
   random       = "whatever you want"

[languages.de]

given_name      = "Derek"
family_name     = "Perkins"
display_name    = "Derek Perkins"
thumbnail       = "img link"
image           = "img link"
short_bio       = "super short bio in German"
bio             = "Here's Derek's awesome bio in German"
email           = "derek@email.com"

[social]
    facebook    = "derekperkins"
    github      = "derekperkins"
    twitter     = "derek_perkins"
    googleplus  = "DerekPerkins1"

[params]
   random       = "whatever you want"
```

**Comment by derekperkins** [comment]
Sorry, I didn't see #2467 before responding, I'll check that out.

**Comment by bep** [comment]
#2467 has nothing to do with this feature. That is "future work".

**Comment by derekperkins** [comment]
I disagree, it's clearly related since something multi-lingual is what broke it in the first place. It doesn't mean that #2467 has to happen right now, but if I'm going to go in and investigate, I'd prefer to at least think about how to handle multi-lingual authors.

**Comment by moorereason** [comment]
Is this issue still relevant to 0.17 or do we need to move this to the 0.18 milestone?

**Comment by derekperkins** [comment]
Can we reopen this issue? I'd like to get it handled now so that #1850 can get merged right after the 0.17 release.
